geps/gep-2648/index.md

```diff
@@ -132,7 +129,35 @@ when a Policy is a Direct Attached Policy:
   of some sort, where the things outside the namespace can opt–out of the behavior.
   (Notably, this is the design that we used for ReferenceGrant).
 
-## Apply Policies to Sections of a Resource
+## Target References
+### Cross Namespace
+
+In all cases, Gateway API policies should only have an effect on the namespace
+they exist within. In the case of policies that could apply to mesh
+implementations, it may be desirable to have a policy that affects traffic
+originating from the local namespace but going to a separate namespace. Unless
+that specific case is desired, all policy target refs should be local and
+exclude the "namespace" field.
+
+### Multiple
+
+In some cases, it may be desirable for a policy to target more than one resource
+at a time. For example, a policy may apply to different variations of what is
+effectively the same Service (store, store-blue, and store-green). If this is
+desired, a policy can choose to support a `targetRefs` list instead of a
+singular `targetRef` field. This list can have a maximum of 16 entries, though
+it may be desirable to start with a lower limit depending on the policy.
+
+#### Migration from Single to Multiple Targets
+
+Existing policies with a single `targetRef` may want to transition to supporting
+multiple `targetRefs`. To accomplish this, we recommending supporting both
+fields in parallel and merging the results, with the `targetRef` field
+effectively becoming prepended to the front of the `targetRefs` list. To ensure
```

Comment:
I don't like this recommendation for the implementation to have to merge two different fields when presented with them. It sets a precedent that we probably don't want to have in future designs.

Reply:
+1,I would just allow one or the other?

Reply:
Good catch, updated, thanks!